lib/main_common.pm

```diff
@@ -537,7 +537,12 @@ sub load_extra_tests {
         }
     }
     else {
-        loadtest "console/zypper_lr_validate";
+        if (check_var('DISTRI', 'sle') && sle_version_at_least('15')) {
```

Comment:
Sorry, this does not make sense to me, especially not with a comment in the file at all why SLE15 should be treated different

Reply:
It was mentioned in lib/utils.pm line 859.

Reply:
L859 is

```perl
    if ($product =~ /IBM-DLPAR-(Adv-Toolchain|SDK|utils)/) {
        my $cmd
          = "zypper lr --uri | awk -F \'|\' -v OFS=\' \' \'{ print \$3,\$4,\$NF }\' | tr -s \' \' | grep --color \"$product\[\[:space:\]\[:punct:\]\[:space:\]\]*$enabled_repo $uri\"";
        run_scripted_command_slow($cmd, slow_type => 2);

```

Sorry, I don't get how that is related.

Reply:
Line 865 (my branch was old):
` # SLES12 does not have 'SLES12-Source-Pool' SCC channel`